### ISSUE TITLE
I can't use VS 2013 (and nuget 2.x) with latest xunit prerelease packages

### ISSUE THREAD

**Issue by TimLovellSmith** [NuGet/Home#896]
I get an error:

Error	1	NuGet Package restore failed for project Microsoft.AzureCache.RoleShared.Tests: System.InvalidOperationException: 'xunit.core' already has a dependency defined for 'xunit.extensibility.core'.
   at NuGet.Manifest.ValidateDependencySets(IPackageMetadata metadata)
   at NuGet.Manifest.Validate(Manifest manifest)
   at NuGet.Manifest.ReadFrom(Stream stream, IPropertyProvider propertyProvider, Boolean validateSchema)
   at NuGet.LocalPackage.ReadManifest(Stream manifestStream)
   at NuGet.OptimizedZipPackage.EnsureManifest()
   at NuGet.OptimizedZipPackage..ctor(IFileSystem fileSystem, String packagePath, IFileSystem expandedFileSystem)
   at NuGet.SharedPackageRepository.OpenPackage(String path)
   at NuGet.LocalPackageRepository.GetPackage(Func`2 openPackage, String path)
   at NuGet.LocalPackageRepository.<GetPackages>d__16.MoveNext()
   at NuGet.CollectionExtensions.AddRange[T](ICollection`1 collection, IEnumerable`1 items)
   at NuGet.LocalPackageRepository.FindPackagesById(Func`2 openPackage, String packageId)
   at NuGet.LocalPackageRepository.FindPackagesById(String packageId)
   at NuGet.PackageRepositoryExtensions.FindPackagesById(IPackageRepository repository, String packageId)
   at NuGet.PackageRepositoryExtensions.FindPackage(IPackageRepository repository, String packageId, SemanticVersion version, IPackageConstraintProvider constraintProvider, Boolean allowPrereleaseVersions, Boolean allowUnlisted)
   at NuGet.InstallWalker.ResolveOperations(IPackage package)
   at NuGet.PackageManager.Execute(IPackage package, IPackageOperationResolver resolver)
   at NuGet.PackageManager.InstallPackage(IPackage package, FrameworkName targetFramework, Boolean ignoreDependencies, Boolean allowPrereleaseVersions, Boolean ignoreWalkInfo)
   at NuGet.PackageManager.InstallPackage(IPackage package, Boolean ignoreDependencies, Boolean allowPrereleaseVersions, Boolean ignoreWalkInfo)
   at NuGet.Common.PackageExtractor.<>c__DisplayClass0.<InstallPackage>b__2()
   at NuGet.Common.PackageExtractor.ExecuteLocked(String name, Action action)
   at NuGet.Common.PackageExtractor.InstallPackage(IPackageManager packageManager, IPackage package)
   at NuGet.VsEvents.PackageRestorer.RestorePackage(PackageReference package)
   at NuGet.VsEvents.PackageRestorer.RestorePackages(String packageReferenceFileFullPath, IFileSystem fileSystem)
   at NuGet.VsEvents.PackageRestorer.PackageRestore(ProjectPackageReferenceFile projectPackageReferenceFile).		0	0	

This error doesn't happen to me with VS 2015.

**Comment by TimLovellSmith** [comment]
In this scenario I've done an incomplete update to xunit 2.0 across my solution. Some projects are looking like this in packages.config

```
<?xml version="1.0" encoding="utf-8"?>
<packages>
  <package id="FakeItEasy" version="2.0.0-beta009" targetFramework="net45" />
  <package id="Microsoft.Bcl" version="1.1.9" targetFramework="net45" />
  <package id="Microsoft.Bcl.Async" version="1.0.168" targetFramework="net45" />
  <package id="Microsoft.Bcl.Build" version="1.0.21" targetFramework="net45" />
  <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net45" />
  <package id="xunit" version="2.0.0" targetFramework="net45" />
  <package id="xunit.abstractions" version="2.0.0" targetFramework="net45" />
  <package id="xunit.assert" version="2.0.0" targetFramework="net45" />
  <package id="xunit.core" version="2.1.0-beta3-build3029" targetFramework="net45" />
  <package id="xunit.extensibility.core" version="2.0.0" targetFramework="net45" />
</packages>
```

whereas others are looking like this:

```
<packages>
  <package id="FakeItEasy" version="2.0.0-beta009" targetFramework="net45" />
  <package id="Microsoft.AspNet.WebApi" version="5.2.2" targetFramework="net45" />
  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.2" targetFramework="net45" />
  <package id="Microsoft.AspNet.WebApi.Core" version="5.2.2" targetFramework="net45" />
  <package id="Microsoft.AspNet.WebApi.WebHost" version="5.2.2" targetFramework="net45" />
  <package id="Newtonsoft.Json" version="6.0.4" targetFramework="net45" />
  <package id="xunit" version="2.0.0" targetFramework="net45" />
  <package id="xunit.abstractions" version="2.0.0" targetFramework="net45" />
  <package id="xunit.assert" version="2.0.0" targetFramework="net45" />
  <package id="xunit.core" version="2.0.0" targetFramework="net45" />
  <package id="xunit.extensibility.core" version="2.0.0" targetFramework="net45" />
</packages>
```

The errors are appearing for the projects with the older xunit version.

**Comment by TimLovellSmith** [comment]
When I try to get out of this state with package manager console, I run into more problems:

PM> Update-Package xunit -Pre
Update-Package : 'xunit.core' already has a dependency defined for 'xunit.extensibility.core'.
Operation is not valid due to the current state of the object.
PM>

**Comment by TimLovellSmith** [comment]
Fixed by installing 2.8.60318.734 from the VS extensions/updates in VS.
